README.md

```diff
@@ -333,48 +339,54 @@ public interface ExampleHandler {
   // ...
 }
 ```
-You can declare your own methods that will be called in `onInvoke()` method of basic ManualHandler.
+You can declare your own methods that will be called in `onInvoke()` method of
+basic ManualHandler.
 
-##### Named
+##### NotNull
 
-Gets the field of the received packet by specified name. Can be multi-level.
+Checks if the needed value is not null before calling the handler method
+(in case the value is null the method will be not called).
 
 ```java
 @JSTPHandler
 public interface ExampleHandler {
   // ...
   @NotNull
-  @Named("ok")
-  void onOK(JSArray args);
-
-  @NotNull
-  @Named("error")
-  void onError(JSArray args);
-
-  // gets value by key "neededValue" in object got by "ok"
-  @Named(value = {"ok", "neededValue"})
-  void onNeededValueGot(String value);
+  void onExampleValue(JSArray args);
   // ...
 }
 ```
 
-##### NotNull
+##### Named
 
-Checks if the needed value is not null before calling the handler method.
+Gets the field of the received packet by specified name. It also allows getting
```

Comment:
'It also allows getting elements from nested obj, the value will be retrieved in the order of keys specified.'
Thanks to @LeraSavchenko.